### PR TITLE
[TS] [Urgent] LPS-190214

### PR DESCRIPTION
**Pull request by marianoalvarosaiz**

### CODE REVIEW

**Comment by Kvale1733** [comment]
ci:test:stable

**Comment by Kvale1733** [comment]
ci:test:um

**Comment by Kvale1733** [comment]
ci:test:blogs

**Comment by Kvale1733** [comment]
@marianoalvarosaiz No regressions found.

File: modules/apps/commerce/commerce-initializer-util/src/main/java/com/liferay/commerce/initializer/util/internal/osgi/commands/CommerceOSGiCommands.java [hunk 1]
```diff
@@ -60,9 +58,9 @@ public void initializeSite(long groupId, String key) throws Exception {
 		Role role = _roleLocalService.fetchRole(
 			company.getCompanyId(), RoleConstants.ADMINISTRATOR);
 
-		List<User> roleUsers = _userLocalService.getRoleUsers(role.getRoleId());
+		long[] userIds = _userLocalService.getRoleUserIds(role.getRoleId());
 
-		User user = roleUsers.get(0);
+		User user = _userLocalService.getUser(userIds[0]);
```
Comment (tinatian) [inline modules/apps/commerce/commerce-initializer-util/src/main/java/com/liferay/commerce/initializer/util/internal/osgi/commands/CommerceOSGiCommands.java:63]: @marianoalvarosaiz This does not make sense to me.

Before, there is only invocation to _userLocalService.
After, there are two invocations to _userLocalService.

How do you know "after" is performing better than "before"?

**Comment by tinatian** [comment]
@marianoalvarosaiz Please resend.

Thanks.
Tina

**Comment by marianoalvarosaiz** [comment]
See https://github.com/liferay-core-infra/liferay-portal/pull/5143

**Pull request closed without merging**